pkg/operator/configobservation/auth/auth_serviceaccountissuer.go

```diff
@@ -123,6 +127,34 @@ func observedConfig(
 	}, errs
 }
 
+// issuersChanged compares the command line flags used for KAS and the operator status service account issuers.
+// if these two sets are different, we have a change and we need to update the KAS.
+func issuersChanged(kasIssuers []string, activeOperatorIssuer string, trustedOperatorIssuers []string) bool {
+	sort.Strings(kasIssuers)
+	operatorAllIssuers := append([]string{activeOperatorIssuer}, trustedOperatorIssuers...)
+	sort.Strings(operatorAllIssuers)
+	return !sets.NewString(kasIssuers...).Equal(sets.NewString(operatorAllIssuers...))
+}
+
+func getTrustedServiceAccountIssuers(issuers []operatorv1.ServiceAccountIssuerStatus) []string {
```

Comment:
nit: this is both present here and in the other commit in pkg/operator/serviceaccountissuercontroller/serviceaccountissuer_controller.go. It is a trivial func, however common logic. Suggestion is to centralize.

Reply:
@s-urbaniak, on the other hand, if I import the controller, I will make two controllers depend on each-other, which might, in the future, result in import loops. Since it is a simple trivial func, I believe it is fine to copy because the dependency hell is not worth it :-)